lib/main/localizations/app_localizations.dart

```diff
@@ -2487,4 +2487,34 @@ class AppLocalizations {
         'Older',
         name: 'older');
   }
+
+  String get storageQuotas {
+    return Intl.message(
+      'Storage',
+      name: 'storageQuotas',
+    );
+  }
+
+  String textQuotasUsed(num used, num softLimit) {
+    return Intl.message(
+      '$used GB of $softLimit GB Used',
+      name: 'textQuotasUsed',
+      args: [used, softLimit],
+    );
+  }
+
+  String textQuotasWarningTitle(num progress) {
+    return Intl.message(
+      'You are running out of storage ($progress%).',
+      name: 'textQuotasWarningTitle',
+      args: [progress],
+    );
+  }
+
+  String get textQuotasWarningContent {
+    return Intl.message(
+      "Soon you won't be able to email in Tmail. Please clean your storage or upgrade your storage to get full features in Tmail.",
```

Comment:
Please note that, from right now, it should be `Team Mail`, some designs are not up-to-date, but please change it during your work